src/main/resources/view/Fonts.css

```diff
@@ -0,0 +1,14 @@
+@font-face {
+    font-family: "Open Sans";
+    src: url("/fonts/open-sans/OpenSans-Regular.ttf");
+}
+
+@font-face {
+    font-family: "Open Sans SemiBold";
+    src: url("/fonts/open-sans/OpenSans-SemiBold.ttf");
+}
+
+@font-face {
+    font-family: "Open Sans Light";
+    src: url("/fonts/open-sans/OpenSans-Light.ttf");
+}
```

Comment:
Now that we also include the other font weights, should we also define them here? One concern is definitely the definition of unused entities, but on the other hand other developers trying to tweak font weights might be confused why some weights work and some doesn't especially if they are not familiar with javafx and css. An arguably better alternative is to mention it in the developer guide to provide some hints.

Reply:
I think it would be better to include the other font weights, and leave a comment that they are currently unused in AB-3. Perhaps this is out of scope of the developer guide as that currently pertains to the architecture and only mentions JavaFX in passing?

Reply:
I agree with @JoelHo on this issue. Shall I add in the other font weights but leave the code commented out? So it becomes clearer to the students that the rest of the weights are not being used.

Reply:
Leaving unused code commented out looks like a bad practice that shouldn't be encouraged though, while not in the scope of CS2103, [YAGNI](https://en.wikipedia.org/wiki/You_aren't_gonna_need_it) is perhaps something we could take into account.

Reply:
I'm OK to add them here, for completeness sake.